### ISSUE TITLE
Upload not allowed to this directory

### ISSUE THREAD

**Issue by tonycody** [svenstaro/miniserve#1263]
```yaml
version: "3.8"
services:
  miniserve:
    image: svenstaro/miniserve:0.24.0-alpine
    container_name: miniserve
    restart: always
    ports:
      - "8080:8080"
    command: /app/data/
    environment:
      - MINISERVE_TITLE=玩蜂文件共享服务
      - MINISERVE_MKDIR_ENABLED=true
      - MINISERVE_ALLOWED_UPLOAD_DIR=true
      - MINISERVE_HIDE_VERSION_FOOTER=true
      - MINISERVE_SHOW_WGET_FOOTER=true
      - MINISERVE_DIRS_FIRST=true
      - MINISERVE_README=true
    volumes:
      - /opt/miniserve/data:/app/data:rw
```
![image](https://github.com/svenstaro/miniserve/assets/6773121/2d9445b6-caca-4378-969b-6b0c85c2a1a0)
![image](https://github.com/svenstaro/miniserve/assets/6773121/056113eb-6ebf-42fc-8dbf-818c2baa2864)
![image](https://github.com/svenstaro/miniserve/assets/6773121/3233a8ea-da5b-4b91-8366-bb3f819a9397)

**Comment by svenstaro** [comment]
If you use environment variables, you'll have to always specify a full path currently. I'd accept a PR to change that.

**Comment by tonycody** [comment]
您的来信我已收到，我会尽快回复您！！！

    ------------------------------------------------------

**Comment by svenstaro** [comment]
This should be fixed as of miniserve v0.25.0. Please re-open if it isn't.
